classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  attr_accessor :label
+  attr_reader :students
+
+  def add_student(stud)
+    @students.push(stud) 
+  end
```

Comment:
[OPTIONAL] The method that serves as the 'has-many' side (a classroom has many students) is still not completed yet. You must also assign the object itself to the `stud`:
```rb
def add_student(stud)
  @students.push(stud)
  stud.classroom = self
end
```
Check out this article with examples: [both-ways relationships](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)